python/tank/authentication/shotgun_shared/saml2_sso.py

```diff
@@ -248,20 +272,22 @@ def start_sso_renewal(self):
 
             # For debugging purposes
             # @TODO: Find a better way than to use this EV
-            if "SAML2_SHOTGUN_USING_VM" in os.environ:
-                interval = 5000
+            if "SHOTGUN_SSO_DEVELOPER_ENABLED" in os.environ:
```

Comment:
Could this be changed to an environment variable that allows to set the renewal instead of a flag that allows to set it to a fixed time? Could this be useful for clients who want to debug something? Or do you foresee people not having to care about what the interval is?